api/src/repositories/bcgw-repository.ts

```diff
@@ -0,0 +1,43 @@
+import { Geometry } from 'geojson';
+import { z } from 'zod';
+import { getKnex } from '../database/db';
+import { ApiExecuteSQLError } from '../errors/api-error';
+import { Srid } from '../services/geo-service';
+import { BaseRepository } from './base-repository';
+
+export class BCGWRepository extends BaseRepository {
```

Comment:
Could you add a comment to this class?
 - What is BCGW?
 - Add any relevant links?

Reply:
I saw this was described pretty well in the service file. Maybe copy a comment from there?

Reply:
found out during development we didn't need these files so I just removed them